setup.py

```diff
@@ -8,7 +8,7 @@
     long_description = f.read()
 
 # Parse the version from the mapbox module.
-with open('mapbox/__init__.py') as f:
+with open('mapbox/about.py') as f:
```

Comment:
I stumbled onto a piece of advice at https://packaging.python.org/guides/single-sourcing-package-version/ (see #3). Our current practice, #6 on that page has a potential weakness.